content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -74,12 +84,14 @@ For example:
    }
    ```
 
-2. Starting from resource version 10245, receive notifications of any creates, deletes, or updates as individual JSON objects.
+2. Starting from resource version 10245, receive notifications of any creates, deletes, or updates as a concatenated stream of JSON documents.
 
    ```
-   GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245
-   ---
-   200 OK
+   GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245 HTTP/1.1
+   …more headers…
```

Comment:
I would not include line 91, `…more headers…`

Reply:
Here's a thing though. If someone who doesn't know HTTP sees that, will they realize it's only part of a valid request?

Reply:
I'm not aware of any conventions for representing HTTP request and response. I'm not convinced that line 90 will be helpful for readers in this context.

Reply:
I've omitted this change from the latest revision.